### ISSUE TITLE
[BottomAppBar] Floating button doesn't read out as a button after the position changes

### ISSUE THREAD

**Issue by codeman7** [material-components/material-components-ios#4563]
If you set the accessibility label for the floatingButton and then change it's position it will no longer read out that the FAB is a button. Instead it just reads the label.

## Reproduction steps
1. Set an accessibilityLabel for the floatingButton
2. Turn on VoiceOver
3. Open Catalog
4. Go to BottomAppBar example
5. Navigate to the FAB
6. Double tap to move the FAB

## Expected behavior

Every time VoiceOver reads the FAB it should read that it is a button as well.

## Actual behavior

Only reads that it is a button before moving it's position.

## Platform (please complete the following information)

 - Device: iPhoneX
 - OS: iOS 11

**Comment by romoore** [comment]
I was unable to reproduce this on an iPad Mini4/iOS 10.3.2 nor on an iPhone X/iOS 11.4 (simulator). Here are the steps I performed. I installed develop/120c93d

1. In BottomAppBarTypicalUseSupplemental.m/-viewWillAppear I set the Bottom App Bar's FAB `accessibilityLabel` to "Hello world"
1. Turned on VoiceOver
1. I launched the catalog and navigated to the example
1. I selected the FAB and it read-out the label and trait (button)
1. I activated the FAB and it disappeared
1. I toggled the "Visible FAB" switch to make the FAB reappear
1. I selected the FAB and it read-out the label and trait. The Accessibility Inspector also showed both the label (title) and trait appearing correctly.

Please reopen this issue with steps to reproduce if you are able.
